client/client.go

```diff
@@ -674,6 +673,8 @@ func (c *Client) downloadMetaFromTimestamp(name string, m data.TimestampFileMeta
 }
 
 // decodeRoot decodes and verifies root metadata.
+//
+//lint:ignore U1000 unused
```

Comment:
worth removing these unused functions? i can't remember the context.

Reply:
We probably should. Done.